src/Admin/ElectedRepresentativeAdmin.php

```diff
@@ -151,6 +162,11 @@ protected function configureShowFields(ShowMapper $showMapper)
 
     protected function configureFormFields(FormMapper $formMapper)
     {
+        $mandates = [];
+        if (null !== $this->getSubject()) {
+            $mandates = $this->mandateRepository->getMandatesForPoliticalFunction($this->getSubject());
+        }
+
```

Comment:
what about:
```suggestion
        $mandates = [];
        
        $electedRepresentative = $this->getSubject();
        if ($electedRepresentative instanceof ElectedRepresentative) {
            $mandates = $this->mandateRepository->getMandatesForPoliticalFunction($electedRepresentative);
        }

```

this will make your IDE understand you are passing a valid parameter to the `MandateRepository::getMandatesForPoliticalFunction()` method